test/integration/base.py

```diff
@@ -26,30 +23,11 @@ def set_http_debug(self):
             http.client.HTTPConnection.debuglevel = 1
 
     @pytest.fixture(autouse=True)
-    def save_settings(self, dont_cleanup_old_buckets, b2_auth_data):
-        type(self).dont_cleanup_old_buckets = dont_cleanup_old_buckets
```

Comment:
I believe that this flag had some logic behind it. Could you tell me, in short, if the new code "has something similar" or was it just not needed, please?

Reply:
having changed the logic for removing bucket (using mixin) it seem to me that we no longer need to keep this flag